kt/godot-library/src/main/kotlin/godot/core/Color.kt

```diff
@@ -535,6 +587,237 @@ class Color(
             val r = (hex and 0xFFFF) / 65535.0
             return Color(r, g, b, a)
         }
+
+        private val namedColors = arrayOf(
```

Comment:
Should add more consistency here. Those names already exist as variables, meaning we create them two times in two different ways (one with constructor and one with hex). Should be merged so we only create them one time.